src/commands/token/exchangeTokenCommand.ts

```diff
@@ -0,0 +1,47 @@
+interface ExchangeTokenInterface {
+    grant_type: string;
+    redirect_uri?: string | undefined;
+    client_id: string;
+    client_secret: string;
+    code: string;
+}
+export default class ExchangeTokenCommand implements ExchangeTokenInterface {
+    readonly grant_type: string;
+    readonly client_id: string;
+    readonly client_secret: string;
+    readonly code: string;
+    readonly redirect_uri: string | undefined;
```

Comment:
if `redirect_uri` is undefined we should mark it optional and remove the `| undefined`

Reply:
Yes !